docs/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Double checking, but adding the Gatsby license was intentional yes? Because:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
totally not intentional